workitem/workitemtype.go

```diff
@@ -101,8 +102,14 @@ type WorkItemType struct {
 	// the DB. Instead the Path member contains the information.
 	Extends uuid.UUID `gorm:"-" json:"extends,omitempty"`
 
-	// CanConstruct is true when you can create work items from this work item
```

Comment:
Restore not remove the `CanConstruct` comment please.

Reply:
Fixed. 4a7dc79.